github-org/README.md

```diff
@@ -0,0 +1,36 @@
+# Mondoo GitHub Organization Action
+
+A GitHub Action for using Mondoo to scan a GitHub Organization for security vulnerabilities and misconfigurations. This Action can be easily used in [.github or .github-private](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile) repositories.
+
+## Properties
+
+The GitHub Organization Action has properties which are passed to the underlying image. These are passed to the action using `with`.
+
+| Property                      | Required | Default | Description                                                                                                                                                          |
+| ----------------------------- | -------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `organization`                | true     |         | GitHub organization to scan eg. `mondoohq` Client.                                                                                                                   |
+| `token`                       | true     |         | GitHub token used for authentication                                                                                                                                 |
```

Comment:
Can we document the privileges required by Mondoo to scan an Org? It would be great to make this clear. I believe we just need `org:read` but I am not totally sure yet. Do you know @chris-rock ? 🤔

Reply:
We should totally do that